### ISSUE TITLE
Can we install nginx staff in a standalone container?

### ISSUE THREAD

**Issue by hbin** [seapy/dockerfiles#3]
It seem like that you build all the staff in one image. But as the official `Best Practices` says: [Run only one process per container](https://docs.docker.com/articles/dockerfile_best-practices/#run-only-one-process-per-container).

**Comment by seapy** [comment]
you're right, i know that. but this docker image(rails-nginx-unicorn-*) help easy to start rails app.

if split nginx, create new Dockerfile(like rails-unicorn) only execute unicorn and link with nginx container.. link strategy is depends on application, currently i don't have good idea. someday i will do that.

thankyou

**Comment by itsSaad** [comment]
maybe a docker-compose can be added

**Comment by seapy** [comment]
@itsSaad I think the same as you.
but docker compose is not recommend production yet.

**Comment by itsSaad** [comment]
@seapy do you happen to know what really is wrong with using it in production. Docker guide only tells us not to use in production. why not?

**Comment by laughingMan** [comment]
http://www.infoworld.com/article/3046464/application-development/docker-not-production-ready-not-so-says-docker.html
